### PR TITLE
feat: migrate check

### PR DESCRIPTION
**Pull request by zhangsoledad**
`ckb migrate --check`
if migration is in need 0 will be return，otherwise ExitCode::Cli(64) returned.

### CODE REVIEW

File: util/app-config/src/cli.rs [hunk 1]
```diff
@@ -245,7 +246,13 @@ fn import() -> App<'static, 'static> {
 }
 
 fn migrate() -> App<'static, 'static> {
-    SubCommand::with_name(CMD_MIGRATE).about("Runs ckb migration")
+    SubCommand::with_name(CMD_MIGRATE)
+        .about("Runs ckb migration")
+        .arg(
+            Arg::with_name(ARG_MIGRATE_CHECK)
+                .long(ARG_MIGRATE_CHECK)
+                .help("Perform database version check without migrating"),
```
Comment (doitian) [inline util/app-config/src/cli.rs:254]: Please add detailed document about the exist code: what does exit code 0 means.

**Review by doitian — CHANGES_REQUESTED** [review]

File: util/app-config/src/cli.rs (hunk 1 above)
Comment (zhangsoledad) [reply util/app-config/src/cli.rs]: done. :haircut_man:

**Review by yangby-cryptape — DISMISSED** [review]

**Review by quake — DISMISSED** [review]

**Review by doitian — DISMISSED** [review]

**Review by doitian — APPROVED** [review]

**Review by yangby-cryptape — APPROVED** [review]

**Comment by doitian** [comment]
bors r=doitian,yangby-cryptape

**Review by doitian — APPROVED** [review]

**Review by yangby-cryptape — APPROVED** [review]

**Comment by doitian** [comment]
bors r=doitian,yangby-cryptape

**Pull request merged**